### PR TITLE
feat(mux-uploader / mux-uploader-react): Add UpChunk's dynamicChunkSize

### PR DESCRIPTION
**Pull request by decepulis**

### CODE REVIEW

File: packages/mux-uploader/package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "publish-release": "../../scripts/publish.sh"
   },
   "dependencies": {
-    "@mux/upchunk": "^2.3.1"
+    "@mux/upchunk": "^2.5.0"
```
Comment (cjpillsbury) [inline packages/mux-uploader/package.json:49]: todo(blocker): This should be upgraded to 2.6.0, since there was another release cut.

**Review by cjpillsbury — CHANGES_REQUESTED** [review]
One minor todo (another version bump). Otherwise, 😎

File: packages/mux-uploader/package.json (hunk 1 above)
Comment (decepulis) [reply packages/mux-uploader/package.json]: ✅

**Review by cjpillsbury — APPROVED** [review]
🎉

**Pull request merged**